docker/compose/docker-compose-extendeds3.yml

```diff
@@ -10,7 +10,7 @@
 version: '2'
 services:
   zookeeper:
-    image: zookeeper:3.5.4-beta
+    image: zookeeper:3.6.1
```

Comment:
I thought that we were on 3.5.5, but maybe my understanding is outdated already.

Reply:
I checked this and noticed it was `3.6.1` now. 

https://github.com/pravega/zookeeper-operator/blob/db0e7467853123ed75053633d859b246dcefaf05/docker/Dockerfile#L17

Reply:
`gradle.properties` still says `3.5.5`.

Reply:
Upgrading Zookeper client will require code changes in Pravega (in the class below).  In this PR, I've only changed the Zookeeper server version in templates to reflect the same version used in Zookeeper operator. 
```
pravega/segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/ZooKeeperServiceRunner.java
```
It is not the intent of this PR to upgrade the Zookeeper client and I think we should do it separately.